bin/record.js

```diff
@@ -16,6 +16,8 @@ const MOVIE_REGEX = /🎥/g
 
 const ffmpegPath = require('@ffmpeg-installer/ffmpeg').path
 
+const recordFailedTests = process.env.RECORD_FAILED_TESTS
```

Comment:
process the environment variable here to get a boolean variable. I would do something like
```js
let recordFailedTests = false
if ('RECORD_FAILED_TESTS' in process.env) {
  recordFailedTests = process.env.RECORD_FAILED_TESTS === 'true' || process.env.RECORD_FAILED_TESTS === '1'
}
```